docs/integrations/destinations/pinecone.md

```diff
@@ -0,0 +1,42 @@
+# Pinecone
+
+## Overview
+
+This destination prepares data to be used by [Pinecone](https://pinecone.io/).
+
+There are three parts to this:
+* Processing - split up individual records in chunks so they will fit the context window and decide which fields to use as context and which are supplementary metadata.
+* Embedding - convert the text into a vector representation using a pre-trained model (Currently, OpenAI's `text-embedding-ada-002` and Cohere's `embed-english-light-v2.0` are supported.)
+* Indexing - store the vectors in a vector database for similarity search
+
+### Processing
+
+Each record will be split into text fields and meta fields as configured in the "Processing" section. All text fields are concatenated into a single string and then split into chunks of configured length. The meta fields are stored as-is along with the embedded text chunks. Please note that meta data fields can only be used for filtering and not for retrieval and have to be of type string, number, boolean (all other values are ignored). Depending on the chosen vector store, additional limitations might apply.
```

Comment:
Mentions "chosen vector store", that doesn't make sense in this context, right? Also we should mention here that meta fields need to be specified as well.